packages/clay-date-picker/src/DateNavigation.tsx

```diff
@@ -30,7 +30,15 @@ type Props = {
 };
 
 const ClayDatePickerDateNavigation = ({
-	ariaLabels,
+	ariaLabels = {
+		buttonChooseDate: 'Choose date',
+		buttonDot: 'Select current date',
+		buttonNextMonth: 'Select the next month',
+		buttonPreviousMonth: 'Select the previous month',
+		chooseDate: 'Use the calendar to choose a Date. Current selection {0}',
+		selectMonth: 'Select a month',
+		selectYear: 'Select a year',
+	},
```

Comment:
Since we already have the predefined values ​​in the root component we don't need to do it here again.